Sources/MongoMobile/MongoMobile.swift

```diff
@@ -53,6 +61,7 @@ private func getStatusExplanation(_ status: OpaquePointer?) -> String {
 public class MongoMobile {
     private static var libraryInstance: OpaquePointer?
     private static var embeddedInstances = [String: OpaquePointer]()
+    private static var embeddedClients = [(WeakRef<MongoClient>, OpaquePointer)]()
```

Comment:
`MongoClient` has a `close` method that calls `mongoc_client_destroy`, so you could just store the `WeakRef<MongoClient>` and  do `ref.reference?.close()`